.travis.yml

```diff
@@ -64,8 +64,17 @@ after_failure: |
     kubectl delete namespace $namespace
   fi
 
+stages:
+  - test
+
 jobs:
   include:
+    - name: Lint for subcharts
+      stage: test
+      script: |
+        helm lint helm/alfresco-content-services/charts/activemq
+        helm lint helm/alfresco-content-services/charts/alfresco-search
```

Comment:
There's also a sub-sub-chart for the insight engine should we lint that as well?

Reply:
Insight-engine is part of search helm chart, you mean insight-zeppelin helm chart?

Reply:
added lint for insight-zeppelin